doc/DISTUTILS.rst.txt

```diff
@@ -361,6 +361,9 @@ during the build process are ready to be compiled. This form
 of generic typing is also supported for C header files (preprocessed to
 produce :file:`.h` files).
 
+Additionally, NumPy distutils supports automatic conversion of source files named *.src". This facility can be used to maintain very similar code blocks requiring only simple changes between blocks. During the build phase of setup, if a template file named <somefile>.src is encountered, a new file named <somefile> is constructed from the template and placed in the build directory to be used instead. Two forms of template conversion are supported. The first form occurs for files named <file>.ext.src where ext is a recognized Fortran extension (f, f90, f95, f77, for, ftn, pyf). The second form is used for all other cases. Read more `here`_.
```

Comment:
The `*.src"` is breaking the build. it should be enclosed in two back-slashes. Same with any non-text code like `<somefile>.src`, `<somefile>`, ...
```
``*.src``
```

Reply:
I'll change it right now